src/WebJobs.Script/Binding/ServiceBusScriptBindingProvider.cs

```diff
@@ -110,8 +112,17 @@ public override Type DefaultType
                 {
                     if (Context.Access == FileAccess.Read)
                     {
-                        return string.Compare("binary", Context.DataType, StringComparison.OrdinalIgnoreCase) == 0
+                        Type type = string.Compare("binary", Context.DataType, StringComparison.OrdinalIgnoreCase) == 0
                             ? typeof(byte[]) : typeof(string);
+
+                        if (string.Compare("many", Context.Cardinality, StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
This is the first binding we've needed this for. Other bindings may also leverage this, if they support array bindings.